pybossa/cache/projects.py

```diff
@@ -630,6 +630,21 @@ def get_from_pro_user():
         projects.append(project)
     return projects
 
+def get_recently_updated_projects():
+    """Return the list of published projects that has task creations in last 3 months."""
+
+    sql = text('''SELECT t.project_id, p.short_name FROM task t, project p
```

Comment:
selecting table1 and table2 is an old syntax. I think we should use explicitly left join or inner join. I know there are many existing code using this syntax, but it is not recommended.

Reply:
ok, will update query.